tests/components/FilterSearch.test.tsx

```diff
@@ -659,7 +659,19 @@ describe('search without section labels', () => {
     await userEvent.type(screen.getByRole('textbox'), 'a');
     expect(mockedOnDropdownInputChange).toHaveBeenCalledTimes(1);
     expect(executeFilterSearch).toHaveBeenCalledTimes(0);
-  })
+  });
+
+  it('when an afterDropdownInputFocus prop is provided, invokes it in addition to the original ' +
+    'behavior when input gains focus', async () => {
+    const mockedaAfterDropdownInputFocus = jest.fn();
+    const executeFilterSearch = jest.spyOn(SearchHeadless.prototype, 'executeFilterSearch');
+    renderFilterSearch(
+      {searchFields: searchFieldsProp, afterDropdownInputFocus: mockedaAfterDropdownInputFocus});
+    await userEvent.type(screen.getByRole('textbox'), 'a');
```

Comment:
it feels weird to type in the input and then click on it. can we first click into the input, check that afterFocus was called, then type a character and check that executeFilterSearch was called?

Reply:
Done